tests/e2e/visual/contacts/list-view-login-visual.wdio-spec.js

```diff
@@ -0,0 +1,132 @@
+const commonPage = require('@page-objects/default/common/common.wdio.page');
+const contactPage = require('@page-objects/default/contacts/contacts.wdio.page');
+const loginPage = require('@page-objects/default/login/login.wdio.page');
+const dataFactory = require('@factories/cht/generate');
+const reportsPage = require('@page-objects/default/reports/reports.wdio.page');
+const utils = require('@utils');
+
+const { resizeWindowForScreenshots, generateScreenshot } = require('@utils/screenshots');
+
+describe('Contact List Page', () => {
+  const updateRolePermissions = async (roleValue, addPermissions, removePermissions = []) => {
+    const roles = [roleValue];
+    const settings = await utils.getSettings();
+    const permissions = await utils.getUpdatedPermissions(roles, addPermissions, removePermissions);
+    await utils.updateSettings(
+      { roles: settings.roles, permissions },
+      { revert: true, ignoreReload: true, refresh: true, sync: true }
+    );
+  };
+
+  const docs = dataFactory.createHierarchy({
+    name: 'Janet Mwangi',
+    user: true,
+    nbrClinics: 10,
+    nbrPersons: 4,
+    useRealNames: true,
+  });
+
+  before(async () => {
+    await resizeWindowForScreenshots();
+    await utils.saveDocs([...docs.places, ...docs.clinics, ...docs.persons, ...docs.reports]);
+    await utils.createUsers([docs.user]);
+  });
+
+  after(async () => {
+    await utils.deleteUsers([docs.user]);
+    await utils.revertDb([/^form:/], true);
+  });
+
+  beforeEach(async () => {
+    await loginPage.login(docs.user);
+  });
+
+  afterEach(async () => {
+    await commonPage.logout();
+  });
+
+  describe('Log in', () => {
+    it('should show contacts page tab '+
+      'when can_view_contact and can_view_contacts_tab permissions are enabled', async () => {
+      await (await commonPage.contactsTab()).waitForDisplayed();
+      await generateScreenshot('contact-page', 'tab-visible');
+      await commonPage.openHamburgerMenu();
+      await generateScreenshot('contact-page', 'menu-opened');
+      await commonPage.closeHamburgerMenu();
+      await commonPage.goToPeople();
+      expect(await commonPage.isPeopleListPresent()).to.be.true;
+      await generateScreenshot('contact-page', 'people-list-visible');
+      await commonPage.goToReports();
+      await reportsPage.openFirstReport();
+      await reportsPage.rightPanelSelectors.patientName().waitForClickable();
+      await generateScreenshot('contact-page', 'reports-visible');
+      await reportsPage.rightPanelSelectors.patientName().click();
+      await contactPage.waitForContactLoaded();
+      await generateScreenshot('contact-page', 'contact-loaded');
+      await commonPage.goToMessages();
+    });
+    it('should hide contacts page as tab and from menu option ' +
```

Comment:
Space between test cases
```suggestion
    });
    
    it('should hide contacts page as tab and from menu option ' +
```